<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableEditor</class>
 <widget class="QWidget" name="TableEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelRow">
       <property name="text">
        <string>Row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRowAddBefore">
       <property name="toolTip">
        <string>Insert row before</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ReportViewer.qrc">
         <normaloff>:/insert-row-before</normaloff>:/insert-row-before</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRowDelete">
       <property name="toolTip">
        <string>Delete row</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ReportViewer.qrc">
         <normaloff>:/delete-row</normaloff>:/delete-row</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRowAddAfter">
       <property name="toolTip">
        <string>Insert row after</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ReportViewer.qrc">
         <normaloff>:/insert-row-after</normaloff>:/insert-row-after</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelColumn">
       <property name="text">
        <string>Column</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonColumnAddBefore">
       <property name="toolTip">
        <string>Insert column before</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ReportViewer.qrc">
         <normaloff>:/insert-column-before</normaloff>:/insert-column-before</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonColumnDelete">
       <property name="toolTip">
        <string>Delete column</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ReportViewer.qrc">
         <normaloff>:/delete-column</normaloff>:/delete-column</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonColumnAddAfter">
       <property name="toolTip">
        <string>Insert column after</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ReportViewer.qrc">
         <normaloff>:/insert-column-after</normaloff>:/insert-column-after</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ReportViewer.qrc"/>
 </resources>
 <connections/>
</ui>
